frontend/lib/features/onboarding/onboarding_welcome_page.dart

```diff
@@ -43,6 +46,22 @@ class OnboardingWelcomePage extends StatelessWidget {
                 child: Text(Loc.of(context).getStarted),
               ),
             ),
+            if (kDebugMode) const SizedBox(height: Grid.md), // Add some spacing
+            if (kDebugMode) Padding(
+              padding: const EdgeInsets.symmetric(horizontal: Grid.side),
+              child: FilledButton(
+                onPressed: () {
+                  Navigator.of(context).push(
+                    MaterialPageRoute(
+                      builder: (context) => const FormsPage(), // Replace with your desired page
+                      fullscreenDialog: true,
+                    ),
+                  );
+                },
+                child: Text('Test Schemas'), // Your custom text
+              ),
+            ),
+            const SizedBox(height: Grid.lg), // Optional spacing at the bottom
```

Comment:
Probably can remove some of these "Chad" comments here ;)

Reply:
what you don't like the widsom of chad?

Reply:
![image](https://github.com/TBD54566975/didpay/assets/14976/5038903c-799e-49d8-a3dc-9321a50116cc)